### DISCUSSION TITLE
Bug: Defined map language not correct for maps & other map-update-related questions

### DISCUSSION THREAD

**Discussion by pstein** [osmandapp/OsmAnd#17321] [General]
1.) I setup in menu
Configure Map--->Map language= "German" as map language

This is working for street names. But when I go to
Download maps--->Europe--->Germany--->

Then the names of some (but curiously not all !!!) of german states maps are in english (e.g. "Bavaria" instead of "Bayern")?
How can I tell OsmAnd to switch to german state names for ALL maps too?

2.) At the top of the map list there is an entry "Standard map".
Whats does that mean? For which underlying map is "standard map" the alias?

3.) Are the already downloaded (offline) maps AUTOMATICALLY updated?
At which frequency?
How can I restrict auto-update of maps to times when I am connected to Wifi (and not mobile provider)?

**Comment by mikehgentry** [comment]
2) 'Standard maps' just means vector maps, rather than (raster) tiles

3) Vector maps never get updated automatically - you must go to 'Download maps' -> 'Updates' to update them.

Tile maps will get automatically updated according to the frequency set for tile expiry in the 'Download maps' -> 'Local' tab. By long-pressing on a point on the map and choosing 'Actions' you can manually update tiles when you're on wifi to avoid this.

There have been a few issues discussing ways to make it work better, e.g. #15187
